__test_utils__/TestInput.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { withFormsy } from './..';
+import { PassDownProps } from '../src/Wrapper';
 
-class TestInput extends React.Component {
-  static defaultProps = { type: 'text' };
-
+class TestInput extends React.Component<React.HTMLProps<HTMLInputElement>> {
   updateValue = event => {
-    this.props.setValue(event.target[this.props.type === 'checkbox' ? 'checked' : 'value']);
+    const formsyProps = this.props as PassDownProps;
+    formsyProps.setValue(event.target[this.props.type === 'checkbox' ? 'checked' : 'value']);
   };
 
   render() {
-    return <input type={this.props.type} value={this.props.value} onChange={this.updateValue} />;
+    return <input type={this.props.type || 'text'} value={this.props.value} onChange={this.updateValue} />;
```

Comment:
These are unrelated typescript changes that I can remove if they feel too out of place for this PR.